service/service_construction.go

```diff
@@ -628,142 +627,109 @@ func (s ConstructionService) CreateOperationDescription(
 		return nil, fmt.Errorf("invalid number of operations")
 	}
 
-	firstCurrency := operations[0].Amount.Currency
-	secondCurrency := operations[1].Amount.Currency
+	currency := operations[0].Amount.Currency
```

Comment:
I'm not sure why we removed the firstCurrency/SecondCurrency to instead keep one broken out and reference the other one within the array

Reply:
After the utils.Equal call on 639, we only need to reference one of them since they're identical